Article 122: ASCII

ImarinArwen opened the pull request:
## Attributes

<!-- Please use one of these items per category by removing the comment tag at the beginning and end of the line. -->

#### Length of text ####
 :page_with_curl: short 
<!-- :book:  long -->

####  Difficulty to read ####
 :mouse:  light
<!-- :elephant:  heavy -->

####  Difficulty to understand #### 
:school_satchel:  easy 
<!-- :mortar_board:  difficult -->

####  Type of text #### 
<!-- :laughing:  funny -->
 :bowtie:  informative 
<!-- :sunglasses:  catchy -->

####  Type of header #### 
<!-- :santa:  funny --> 
<!-- :bulb:  informative -->
 :boom:  catchy

Teifun2 commented:
**Plenum:**
- Not mentioning that we have a world wide convention.
- More than 127? 
- W is uppercase in Text and lowercase in example
- Why the 63?
- Move the example to the image.
- Digit in example.
- Grammar mistakes.

ImarinArwen commented:
- Not mentioning that we have a world wide convention.
Added this sentence: "Today, the international standard for encoding is called Unicode. It is way bigger than ASCII, such that (in some days) every possible character of every language will have its own digital code."

- More than 127?
The first bit was an error correcting code. 2^7 = 128. I thought about mentioning the 8-bit thing and that today the first bit is used as well but I thought this would blow up the text.

- W is uppercase in Text and lowercase in example. 
A: Changed

-    Why the 63? 
A: I wanted the image to look like a riddle. Therefore I wanted the H, *, e on the left and the question mark to be on the right side.
- Digit in example.
A: I will ask Moira. Maybe we will find a solution.

-   Move the example to the image.
 A: I would prefer to leave the example in the text. I wanted that break between the explanation and the general information below.

 - Grammar mistakes.
A: changed the ones I found.

marcelluethi commented on articles/article-122-ascii.md:
```diff
@@ -0,0 +1,36 @@
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+If letters are represented in computers they are stored as numbers. To encode them every letter got its own decimal number between 0 and 127. The "H", for example, is encoded as the 72.
```
Language suggestion: Starting the sentence with if sounds wrong to my ears. Maybe: In computers, letters are represented as numbers.

ImarinArwen replied:
done.

marcelluethi commented on articles/article-122-ascii.md:
```diff
@@ -0,0 +1,36 @@
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+If letters are represented in computers they are stored as numbers. To encode them every letter got its own decimal number between 0 and 127. The "H", for example, is encoded as the 72.
```
Language: To encode them, every letter is assigned its own decimal number between 0 and 127.

ImarinArwen replied:
changed

marcelluethi commented on articles/article-122-ascii.md:
````diff
@@ -0,0 +1,36 @@
+72 101 108 108 111   32  119 111 114 108 100
+```
+
+Since things are stored binary in computers, those numbers are translated in binary code.
````
Language suggestion: As everything is stored in binary in computers ...

ImarinArwen replied:
Changed

marcelluethi commented on articles/article-122-ascii.md:
````diff
@@ -0,0 +1,36 @@
+```
+
+Since things are stored binary in computers, those numbers are translated in binary code.
+In this way are not just the capital letters encoded but also the lowercase letters, the digits 0-9 and punctuation symbols.
````
Language suggestion: Not only the capital letters are encoded in this way, but ....

ImarinArwen replied:
Done.

marcelluethi commented on articles/article-122-ascii.md:
```diff
@@ -0,0 +1,36 @@
+In this way are not just the capital letters encoded but also the lowercase letters, the digits 0-9 and punctuation symbols.
+
+This very common character encoding is called ASCII (American Standard Code for Information Interchange).
+Today, the international standard for encoding is called Unicode. It is way bigger than ASCII, such that (in some days) every possible character of every language will have its own digital code.
```
Language suggestion: It is way bigger than ASCII - big enough that every possible character of every language ...
Question: Not sure that it is necessary to say that this happens in the future. But if you want to say it, I would repharse it as (in some days -> at some point in the future).

ImarinArwen replied:
Changed it to: "It is way bigger than ASCII - big enough that every possible character of every language may have its own digital code."

MoiraZuber commented:
Added the new image.

marcelluethi commented:
Thanks for the changes. Looks good to me now.

noverney commented:
- can just say "is encoded as 72" (don't need "the")
- should add an "are" in the sentence "Not only are the capital letters encoded in this way"

ImarinArwen commented:
Applied the changes.
